config/environment.js

```diff
@@ -150,6 +150,12 @@ module.exports = function (environment) {
       }
     }
 
+    if (process.env.PUBLIC_MODE !== null && process.env.PUBLIC_MODE == 'false') {
```

Comment:
it’s not really a big deal but `!== null` is redundant I believe? I’d write it this way:

`if (process.env.PUBLIC_MODE === 'false')`

Reply:
🤦‍♂️  You would be correct!